tests/dflowfm/test_xyz.py

```diff
@@ -34,8 +44,44 @@ def test_save(self):
         ]
         model = XYZModel(points=points)
 
+        if output_file.is_file():
```

Comment:
What are we doing here? Is this something I should be doing as well in my tests?

Reply:
It deletes the file if it already exists. That makes sure that we are not getting a green testcase that is accidentally caused by an older file still being present. I borrowed it from old code that used to be present in `test_model.py`.

Reply:
Thanks! 👍